sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1035,7 +1035,7 @@ public sealed partial class RedirectPolicy : Azure.Core.Pipeline.HttpPipelinePol
     }
     public partial class RetryPolicy : Azure.Core.Pipeline.HttpPipelinePolicy
     {
-        public RetryPolicy(int maxRetries = 3, Azure.Core.Delay? delay = null) { }
+        public RetryPolicy(int maxRetries = 3, Azure.Core.DelayStrategy? delayStrategy = null) { }
```

Comment:
nit: keep parameter name `delay`?  Totally your call.

Reply:
Oooh - good catch. No strong opinion here, but the other parameters that use `delay` are for TimeSPans.